include/deal.II/grid/tria_accessor.h

```diff
@@ -1296,6 +1296,30 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
   Point<spacedim> intermediate_point(const Point<structdim> &coordinates) const;
 
   /**
+   * This computes an approximate transformation from the real to the unit
+   * cell by inversion of an affine approximation of the $d$-linear function from the
+   * reference $d$-dimensional cell.
+   *
+   * The affine approximation of the unit to real cell approximation is found
+   * by a least squares fit of an affine function to the $2^d$ vertices of the
+   * present object. Due to construction, this transformation does never
+   * invert for any given point if the cell's geometry is valid. On the other
+   * hand, the result of this operation is only an approximation to the
+   * location on the unit cell based on vertex information, ignoring the
+   * attached manifold. The result is only exact in case the transformation
+   * from the unit to the real cell is indeed affine, such as in one dimension
+   * or for Cartesian and affine (parallelogram) meshes.
+   *
+   * For exact transformations to the unit cell, use
+   * Mapping::transform_real_to_unit_cell(), some of which use this method as
+   * an initial guess for a Newton iteration.
```

Comment:
I'd drop the last half sentence -- that's an implementation detail not important to know for users.